moveit_planners/pilz_industrial_motion_planner/src/trajectory_functions.cpp

```diff
@@ -571,8 +573,7 @@ bool pilz_industrial_motion_planner::isStateColliding(const bool test_for_self_c
   collision_detection::CollisionRequest collision_req;
   collision_req.group_name = group->getName();
   collision_detection::CollisionResult collision_res;
```

Comment:
Could you add the verbose argument to this collision request `req.verbose = true`? otherwise, the user gets a `No IK solution found` which is true, but the real reason is that a collision was detected at some point. That should print the links that were found to be in a collision.